README.md

```diff
@@ -0,0 +1,32 @@
+# squashdelta
+Create efficient deltas (patches) between two SquashFS images
+
+## Building from source
+This project uses [autotools](http://inti.sourceforge.net/tutorial/libinti/autotoolsproject.html)
+
+```bash
+$ git clone https://github.com/mgorny/squashdelta.git
+$ cd squashdelta
+```
+
+```bash
+$ aclocal
+$ autoconf
+$ touch AUTHORS NEWS README ChangeLog
+$ mkdir build-aux
+$ autoreconf --install
+$ automake --add-missing
+```
+
+```bash
+$ ./configure --prefix=/opt/bin
```

Comment:
This is a weird prefix to suggest. Please remember that users take this seriously, and they're going to end up installing in this weird location.